### PR TITLE
Replace "emby" with jellyfin, including package name.

### PR DESCRIPTION
**Pull request by bfayers**
Rebranding!

### CODE REVIEW

File: MediaBrowserTv/app/src/main/java/tv/emby/embyatv/presentation/MyImageCardView.java [hunk 1]
```diff
@@ -64,21 +64,21 @@ public MyImageCardView(Context context, AttributeSet attrs, int defStyle, boolea
...
         LayoutInflater inflater = LayoutInflater.from(context);
-        View v = inflater.inflate(tv.emby.embyatv.R.layout.image_card_view, this);
-
-        mImageView = (ImageView) v.findViewById(tv.emby.embyatv.R.id.main_image);
-        mInfoArea = v.findViewById(tv.emby.embyatv.R.id.info_field);
-        mTitleView = (TextView) v.findViewById(tv.emby.embyatv.R.id.title_text);
-        mContentView = (TextView) v.findViewById(tv.emby.embyatv.R.id.content_text);
-        mBadgeImage = (ImageView) v.findViewById(tv.emby.embyatv.R.id.extra_badge);
-        mOverlayName = (TextView) v.findViewById(tv.emby.embyatv.R.id.overlay_text);
+        View v = inflater.inflate(org.jellyfin.androidtv.R.layout.image_card_view, this);
```
Comment (dkanada) [inline MediaBrowserTv/app/src/main/java/tv/emby/embyatv/presentation/MyImageCardView.java]: This should be removed entirely and the resources should be loaded at the beginning of the file with the line `import org.jellyfin.androidtv.R` instead.

File: MediaBrowserTv/app/src/main/res/values-cs/strings.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
```
Comment (dkanada) [inline MediaBrowserTv/app/src/main/res/values-cs/strings.xml]: I would say move the string value changes to their own commit for best practices, makes automated translation services much easier to work with. Weblate and similar programs can really screw things up after a merge.

File: MediaBrowserTv/.idea/gradle.xml [hunk 3]
```diff
@@ -5,22 +5,14 @@
       <GradleProjectSettings>
```
[the remark this answers is not in the document]
Comment (bfayers) [reply MediaBrowserTv/.idea/gradle.xml]: Makes sense. (I'll do this in a seperate pr?)

File: MediaBrowserTv/app/src/main/java/tv/emby/embyatv/browsing/HomeFragment.java [hunk 4]
```diff
@@ -67,14 +66,12 @@
...
         MainTitle = this.getString(R.string.home_title);
-        //Validate the app
-        TvApp.getApplication().validate();
 
         super.onActivityCreated(savedInstanceState);
 
         //Save last login so we can get back proper context on entry
         try {
-            Utils.SaveLoginCredentials(new LogonCredentials(TvApp.getApplication().getApiClient().getServerInfo(), TvApp.getApplication().getCurrentUser()), "tv.emby.lastlogin.json");
+            Utils.SaveLoginCredentials(new LogonCredentials(TvApp.getApplication().getApiClient().getServerInfo(), TvApp.getApplication().getCurrentUser()), "tv.jellyfin.lastlogin.json");
```
[the remark this answers is not in the document]
Comment (bfayers) [reply MediaBrowserTv/app/src/main/java/tv/emby/embyatv/browsing/HomeFragment.java]: That makes sense.

File: MediaBrowserTv/app/src/main/java/tv/emby/embyatv/browsing/HomeFragment.java [hunk 5]
```diff
@@ -108,7 +105,7 @@ public void onClick(DialogInterface dialog, int which) {
 //        new Handler().postDelayed(new Runnable() {
 //            @Override
 //            public void run() {
-//                mActivity.showMessage("Thank You for Testing", "Thank you for helping to test this app.  Please check out the new music functionality and LEAVE FEEDBACK in the testing forum at emby.media/community.",10000);
+//                mActivity.showMessage("Thank You for Testing", "Thank you for helping to test this app.  Please check out the new music functionality and LEAVE FEEDBACK in the testing forum at jellyfin.media/community.",10000);
```
[the remark this answers is not in the document]
Comment (bfayers) [reply MediaBrowserTv/app/src/main/java/tv/emby/embyatv/browsing/HomeFragment.java]: Agreed

File: MediaBrowserTv/app/src/main/java/tv/emby/embyatv/presentation/MyImageCardView.java (hunk 1 above)
Comment (bfayers) [reply MediaBrowserTv/app/src/main/java/tv/emby/embyatv/presentation/MyImageCardView.java]: As in move this stuff up? Such that the top of the file looks like:

```
import org.jellyfin.androidtv.R

View v = inflater.inflate(R.layout.image_card_view, this);

mImageView = (ImageView) v.findViewById(R.id.main_image);
mInfoArea = v.findViewById(R.id.info_field);
mTitleView = (TextView) v.findViewById(R.id.title_text);
mContentView = (TextView) v.findViewById(R.id.content_text);
mBadgeImage = (ImageView) v.findViewById(R.id.extra_badge);
mOverlayName = (TextView) v.findViewById(R.id.overlay_text);

```

File: MediaBrowserTv/app/src/main/res/values-cs/strings.xml (hunk 2 above)
Comment (bfayers) [reply MediaBrowserTv/app/src/main/res/values-cs/strings.xml]: Not really sure how I would?

File: MediaBrowserTv/app/src/main/java/tv/emby/embyatv/presentation/MyImageCardView.java (hunk 1 above)
Comment (dkanada) [reply MediaBrowserTv/app/src/main/java/tv/emby/embyatv/presentation/MyImageCardView.java]: Ah no I wasn't clear enough, just add R to the import section at the beginning of the file so you don't even need to append the package name here.

Comment (bfayers) [reply MediaBrowserTv/app/src/main/java/tv/emby/embyatv/presentation/MyImageCardView.java]: Ah right, just throw the import at the top and change all the lower lines to using `R.` sounds cleaner.

File: MediaBrowserTv/.idea/gradle.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (dkanada) [reply MediaBrowserTv/.idea/gradle.xml]: Yeah that works!

File: MediaBrowserTv/app/src/main/java/tv/emby/embyatv/browsing/HomeFragment.java (hunk 4 above)
[the remark this answers is not in the document]
Comment (bfayers) [reply MediaBrowserTv/app/src/main/java/tv/emby/embyatv/browsing/HomeFragment.java]: @dkanada 76efa56

File: MediaBrowserTv/app/src/main/java/tv/emby/embyatv/browsing/HomeFragment.java (hunk 5 above)
[the remark this answers is not in the document]
Comment (bfayers) [reply MediaBrowserTv/app/src/main/java/tv/emby/embyatv/browsing/HomeFragment.java]: @dkanada 49c48a6

File: MediaBrowserTv/app/src/main/java/tv/emby/embyatv/presentation/CardPresenter.java [hunk 6]
```diff
@@ -280,7 +280,7 @@ protected void resetCardViewImage() {
             //TvApp.getApplication().getLogger().Debug("Resetting card image");
             try {
                 Glide.with(getContext())
-                        .load(Uri.parse("android.resource://tv.emby.embyatv/drawable/loading"))
+                        .load(Uri.parse("android.resource://org.jellyfin.androidtv/drawable/loading"))
```
[the remark this answers is not in the document]
Comment (bfayers) [reply MediaBrowserTv/app/src/main/java/tv/emby/embyatv/presentation/CardPresenter.java]: Looks like people suggested such a thing in 2015. It works, for now.

File: MediaBrowserTv/.idea/gradle.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (dkanada) [reply MediaBrowserTv/.idea/gradle.xml]: done in #8

**Comment by bfayers** [comment]
@thornbill Sorted: 
eaec5fc

File: MediaBrowserTv/app/src/main/res/values-cs/strings.xml (hunk 2 above)
Comment (dkanada) [reply MediaBrowserTv/app/src/main/res/values-cs/strings.xml]: You are right, it would be hard now that it's not the latest commit on your branch. Not need to separate the strings out for this PR.
